### ISSUE TITLE
Add support for Themes

### ISSUE THREAD

**Issue by windsurfer1122** [TheOfficialFloW/NoNpDrm#20]
Original theme licenses are found in ux0:license/theme

**Comment by windsurfer1122** [comment]
@Gnarmagon already did something similar.

**Comment by Gnarmagon** [comment]
I think this would destroy the intention of the flows plugin :/
I mean my "mod" allows to create a fake license of any rif (even PS3,PSP,PSX etc....) without even owning the Game and if he really wanted it that way he would have done a VPK like Homebrew where you press a button to batch create fake licenses instead of a kernel plugin.....

I think he did it as a plugin to don't make it to easy to pirate with his tool but I would like to see if theFlow would allow theme support +1

**Comment by tomazzz369** [comment]
any one tried to install theme, because when i install theme with nonpdrm license after restart vita always restores default theme, theme still works but i need to set theme again

**Comment by Gnarmagon** [comment]
#21 made a PR based on theleechermans mod.

**Comment by Dimensional** [comment]
Sadly that one doesn't work on 3.65

**Comment by Gnarmagon** [comment]
TheLeechermans Version isn't up to date 😉

**Comment by Dimensional** [comment]
Based on what I saw in the source code, wasn't too hard to port to 3.65. After reviewing the updated code, and discovering how blind I was, I managed to copy the necessary lines into the source and compile. Now have it working again. I can't take any credit for the work, as all I did was take someone else's edits and simply copied them into the source like a plagiarist. If necessary, I can release what I compiled, but given how easy it is, I doubt anyone would have any trouble doing it themself.
